src/api/app/views/staging_project/staged_requests/index.xml.builder

```diff
@@ -0,0 +1,5 @@
+xml.staged_requests do
+  @requests.each do |bs_request|
+    xml.request(id: bs_request.number, creator: bs_request.creator, state: bs_request.state, package: bs_request.first_target_package)
```

Comment:
Hm, isn't this confusing to just show the first package name? 
Talking about this: ``bs_request.first_target_package``

Reply:
All release tools only support single action requests - and as such @bgeuken introduced this function for staging. It would actually be good if the API refused to create such a request if the target has a staging workflow.

Reply:
so because o that explicit use case for staging workflows, i.e https://github.com/vpereira/open-build-service/blob/master/src/api/app/models/bs_request.rb#L293-L294, I think the ```bs_request.first_target_package``` is ok @ChrisBr

Reply:
Makes sense, thanks @coolo

Reply:
To be precise, the package (being the first target package of a bs request) was already part of the exposed API. I just refactored the obs factory code a bit and introduced this first_target_package method to reduce code duplication.

Thinking about it, it would probably make more sense to return all target packages. If some client tool (or staging maintainers) decides to only work with requests with a single target package, they could still handle this on their side.
@coolo Or is there some good reason to only support single package requests in a staging workflow (in general, not specific to openSUSE releases).

Reply:
for one we prefer to show package names in backlog or in the staging project overview. That's arguably harder if a request is not a single package. It's one thing to store it in the DB and render it in XML, but it's a different beast to actually show it in UIs. But if you have ideas...